agent/internal/fluent_intg_test.go

```diff
@@ -1,4 +1,5 @@
-//+build integration
+//go:build integration
+// +build integration
```

Comment:
I was under the impression `go:build` superseded the `+build` syntax, mainly because the DSL for `+build` was error prone. In that case, don't we not need both?

Reply:
hmm they mention
> , and all Go files should be updated to have both forms with the same meaning. 

and go fmt does this. maybe we keep them untill most people are on 1.17?

Reply:
if we keep both, people will get to delay upgrading their compiler, so i almost prefer dropping the old one.. but yeah, we can keep both, for the odd case if a user is building determined on their own.

Reply:
IMO it makes sense to go with their recommendation here at least until go 1.17 is a bit more common.